### ISSUE TITLE
browser.helperApps.neverAsk.saveToDisk stops working after download 1 file

### ISSUE THREAD

**Issue by louis57** [mozilla/geckodriver#1291]
## System

* Version: geckodriver v0.20.0
* Platform: Ubuntu 16.04.4 LTS 
* Firefox: v.60.0.1
* Selenium: v3.12.0

## Testcase

I have a test case in which I have to download 3 files, as following steps:
- click button `Download1` to download File1 (a .html file): Download1  HTML `<input value="Download" onclick="self.location.href='clientReportDownload.do'" type="button">`
- click radio button 1 to choose File2
- click button `Download2` to download File2 (a .jpg file): Download2 HTML `<input value="Download" type="SUBMIT">`
- click radio button 2 to choose File3
- click button `Download2` again to download File3 (a .pdf file)

## Problem
I have setup FirefoxProfile with `browser.helperApps.neverAsk.saveToDisk` with a list of mime types. It does work well when I first download File1 (the file is saved automatically to specified path, without opening the Download dialog). But after that, with File2 and File3 it suddenly doesn't work anymore, the Download dialog still appears and the file are not saved. All mime types are listed in mime list.

Below is my code:

```
System.setProperty("webdriver.gecko.driver", pathToGeckoDriver);
String mimeTypes = "application/zip,application/octet-stream,image/jpeg,application/vnd.ms-outlook,text/html,application/pdf";

FirefoxProfile profile = new FirefoxProfile();
profile.setPreference("browser.download.dir", pathToDownload);
profile.setPreference("browser.download.folderList", 2);
profile.setPreference("browser.download.manager.showWhenStarting", false);
profile.setPreference("browser.helperApps.alwaysAsk.force", false);
profile.setPreference("browser.helperApps.neverAsk.saveToDisk", mimeTypes);
profile.setPreference("browser.download.manager.focusWhenStarting",false);
profile.setPreference("browser.download.manager.useWindow", false);
profile.setPreference("browser.download.manager.showAlertOnComplete", false);

FirefoxOptions options = new FirefoxOptions();
options.setProfile(profile);

// create driver
webDriver = new FirefoxDriver(options);
```

I tried with different file types, the first download always works, after that `browser.helperApps.neverAsk.saveToDisk` stops working

**Expected**: `FirefoxProfile setPreference` always works

**Actual**: `FirefoxProfile setPreference` stops working after download 1 file

**Comment by twalpole** [comment]
It's not likely this just stops working.  It is more likely you don't have necessary mime types set in `browser.helperApps.neverAsk.saveToDisk`.  Try adding `image/pjpeg' to the list of mime types and/or check exactly what the mime type being reported is for your jpg file.

**Comment by louis57** [comment]
@twalpole Thanks for looking into the issue. As I described in my report, I tried many standard mime types, but unfortunately, it didn't work. I also tried your suggestion `image/pjpeg`, it didn't work. I also tried some simple plain .txt files, mime type `text/plain` for File2 and File3, it also didn't work. The first download always works.

This is the source where I check mime types [A List of MIME Types by Content Type](https://www.lifewire.com/mime-types-by-content-type-3469108)
